.yaydoc.yml

```diff
@@ -3,26 +3,19 @@ metadata:
   projectname: "Susi Server"
   version: development
 build:
-  theme: sphinx_fossasia_theme   
+  theme:
+    name: sphinx_fossasia_theme
   source: docs
   logo: images/susi-logo-original.svg
   subproject:
     - url: https://github.com/fossasia/susi_android
-      source: docs
     - url: https://github.com/fossasia/chat.susi.ai
-      source: docs
     - url: https://github.com/fossasia/susi_tweetbot
-      source: docs
     - url: https://github.com/fossasia/susi_fbbot
-      source: docs
     - url: https://github.com/fossasia/susi_slackbot
-      source: docs
     - url: https://github.com/fossasia/susi_desktop
-      source: .
```

Comment:
Keep this. The default is `docs`. If you need the source to be `.`, you need to specify it.